hub/scripts.js

```diff
@@ -183,13 +183,24 @@
     return config;
   }
 
+  /**
+   * Change default image to acom default stardand in added in /hub/
+   */
+  function handleImageMetadata() {
+    const shareableMetaImages = document.querySelectorAll('head meta[content *= "/default-meta-image.png"]'); 
+    shareableMetaImages.forEach((el) => {
+        // update the content to point to our default image under /hub/
+        el.setAttribute('content', `${location.host}/hub/default-meta-image.png`);
```

Comment:
you need to add the protocol, maybe use `window.location.origin` or prepend the protocol to your existing code.